packages/wrangler/src/publish.ts

```diff
@@ -1,7 +1,8 @@
 import assert from "node:assert";
 import path from "node:path";
 import { readFile } from "node:fs/promises";
-import esbuild from "esbuild";
+import * as esbuild from "esbuild";
```

Comment:
By the way, this change was necessary because (for some reason) the default was not being picked up when being run inside Jest...